File: sample-conference-videochat/sample-conference-videochat/Components/Settings/SettingsSectionModel.m
//
//  SettingsSectionModel.m
//  sample-conference-videochat
//
//  Created by Injoit on 30/09/15.
//  Copyright © 2015 QuickBlox Team. All rights reserved.
//

#import "SettingsSectionModel.h"

@interface SettingsSectionModel()

@property (copy, nonatomic) NSString *title;
//@property (strong, nonatomic) NSArray *items;

@end

@implementation SettingsSectionModel

+ (instancetype)sectionWithTitle:(NSString *)title items:(NSArray *)items {
    
    SettingsSectionModel *section = [SettingsSectionModel new];
    section.title = title;
    section.items = items;
    
    return section;
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/sample-conference-videochat-swift/Components/Settings/SettingsSectionModel.swift
//
//  SettingsSectionModel.swift
//  sample-conference-videochat-swift
//
//  Created by Vladimir Nybozhinsky on 22.10.2018.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import Foundation

class SettingsSectionModel {
    // MARK: - Properties
    var title = ""
    var items = [BaseItemModel]()
    
    // MARK: - Public Methods
    class func section(withTitle title: String, items: [BaseItemModel]) -> SettingsSectionModel {
        let section = SettingsSectionModel()
        section.title = title
        section.items = items
        return section
    }
}
